### DISCUSSION TITLE
Docs: not found form builder

### DISCUSSION THREAD

**Discussion by valerii15298** [data-driven-forms/react-forms#1337] [Ideas]
**Scope**: Problem with docs, https://data-driven-forms.org/live-editor

I got 404 not found

Yep, it is not related to library itself but if we wanna have a great library we wanna have a great documentation for it as well.
Also as idea make docs a public github repo so that anyone can contribute and make separate issues.

**Comment by rvsia** [comment]
Hello, thank you for reporting this! 

**The documentation is open sourced**: https://github.com/data-driven-forms/react-forms/tree/master/packages/react-renderer-demo

Yeah, the package name is a little bit confusing but it is still coming from times when it was truly a demo of the renderer and since then it evolved. 😅 

I also see an issue with this page: https://data-driven-forms.org/testing

Additionally, the live editor is deprecated and no longer maintained, but we created a replacement here: https://data-driven-forms.org/editor/pro-editor

**Comment by rvsia** [comment]
Fixed the missing file here https://github.com/data-driven-forms/react-forms/pull/1338
